tests/integration/pyramid_tests.py

```diff
@@ -265,12 +265,14 @@ def test_exception_in_exception_view_caught(self):
     @mock.patch("random.getrandbits")
     def test_distrust_headers(self, getrandbits):
         getrandbits.return_value = 1234
-        self.baseplate_configurator.trust_trace_headers = False
+        self.baseplate_configurator.header_trust_handler.trust_headers = False
 
         self.test_app.get("/example", headers={
             "X-Trace": "1234",
             "X-Parent": "2345",
             "X-Span": "3456",
+            "X-Sampled": "1",
```

Comment:
i think these can be dropped now right? (with the patches on master to fix the issue that made 'em required)

Reply:
Yep, I'll remove it for an extra check on that logic.